config/jobs/periodic/cluster-api-provider-ibmcloud/test-e2e-capi-ibmcloud-periodics.yaml

```diff
@@ -0,0 +1,28 @@
+periodics:
+  - name: periodic-cluster-api-provider-ibmcoud-e2e-test
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    decorate: true
+    interval: 12h
```

Comment:
can we run more often in the beginning like once in 3h to understand the trend? later we can switch with 12h

Reply:
yes, that would be good.